osx/KA-Lite/KA-Lite/README.md

```diff
@@ -1,41 +1,34 @@
 KA-Lite OS X App
 ========================
 
-This is the KA-Lite status menu app with the source and PyRun in one package.
-
 To install:
 
-* Drag the "KA-Lite" app into the "Applications" folder.
+* Double click the "KA-Lite.pkg" and follow the steps of the setup wizard.
 
 
 ## Features
 
 1. Show status icon at system status menu.
-1. Preferences dialog to customize admin account and run the setup process.
-1. Makes the `kalite` executable runnable from anywhere on the terminal.
```

Comment:
Why did you remove the feature about "making the `kalite` executable from anywhere in Terminal"?